Client/mods/deathmatch/logic/CClientVectorGraphic.h

```diff
@@ -53,5 +59,7 @@ class CClientVectorGraphic final : public CClientTexture
 
     std::unique_ptr<CClientVectorGraphicDisplay> m_pVectorGraphicDisplay;
 
+    std::variant<CLuaFunctionRef, bool> m_updateCallbackRef = false;
```

Comment:
std::optional? You don't seem to be using = true

Reply:
brain fart, will update